src/pages/Possibility/KivanSlider.vue

```diff
@@ -0,0 +1,371 @@
+<template>
+	<kv-carousel
+		:autoplay="false"
+		ref="KvCarousel"
+		@change="slideTransition($event)"
+	>
+		<div class="slide-placeholder"></div>
+		<div class="bullet-wrap text-center">
+			<div
+				v-for="(slide , index) in slidesData"
+				:key="index"
+				:class="['bullet', {active: currentIndex === index}]"
+				@click="setSlide(index)"
+			></div>
+		</div>
+		<button @click="previousSlide">
```

Comment:
I think it would be good to have these 2 buttons be big fat hit targets. Probably fill the whole height of the carousel and have a little extra width. Also we should add :hover, :active, and :focus styling.